src/test/java/org/kiwiproject/registry/eureka/client/EurekaRegistryClientTest.java

```diff
@@ -46,14 +45,14 @@ void setUp() {
         var realClient = new EurekaRegistryClient(config, new EurekaRestClient());
         client = spy(realClient);
 
-        EUREKA.getEurekaServer().registerApplication(InstanceInfo.Builder.newBuilder()
-                .setAppName("TEST-APP")
-                .setVIPAddress("TEST-APP")
-                .setHostName("localhost")
-                .setInstanceId("localhost")
-                .setStatus(InstanceInfo.InstanceStatus.UP)
-                .setLeaseInfo(LeaseInfo.Builder.newBuilder().build())
-                .build());
+        EUREKA.getEurekaServer()
+                .getRegistry()
+                .registerApplication("TEST-APP", "localhost", "TEST-APP", "UP");
+    }
+
+    @AfterEach
+    void cleanupEureka() {
+        EUREKA.getEurekaServer().getRegistry().cleanupApps();
```

Comment:
I thought you were going to add a nice method to make this just `EUREKA.cleanupApps()` ? Also, perhaps "cleanup" is not the correct word. Doesn't it really mean to  _remove_ all the registered apps?

Reply:
That will require a change to the embedded-eureka which would cause another release.

Reply:
Created kiwiproject/embedded-eureka#41 to address this